agent/stats/container.go

```diff
@@ -65,6 +67,7 @@ func (container *StatsContainer) StopStatsCollection() {
 
 func (container *StatsContainer) collect() {
 	dockerID := container.containerMetadata.DockerID
+	backoff := retry.NewExponentialBackoff(time.Second*1, time.Second*20, 0.75, 2)
```

Comment:
these values should be constants. also how did we come up with these?

Reply:
why constants if they're only used in one place?

Reply:
more for readability.

Reply:
The numbers are basically arbitrary. Jitter is relatively high as I wanted to make sure that many containers don't synchronize on reopening stats streamer at the same time.

Reply:
ok, lets see from testing how this plays up with the 20 second metrics publish interval to ECS backend service